src/praisonai/praisonai/cli/commands/gateway.py

```
@@ -192,6 +192,102 @@ def gateway_channels(
+            rest_url += "/"
+        
+        response = requests.post(f"{rest_url}api/channels/{name}/pause", timeout=10)
+        response.raise_for_status()
+        
+        result = response.json()
+        if result.get("success"):
+            print(f"✅ Channel '{name}' paused successfully")
+        else:
+            print(f"❌ Failed to pause channel '{name}': {result.get('error', 'Unknown error')}")
+    
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)